catalog/source/workflows.yml

```diff
@@ -463,10 +529,10 @@ workflows:
       - key: gtf
         variable: GENE_MODEL_URL
       - key: fastq PE collection
+        variable: SANGER_READ_RUN_PAIRED
```

Comment:
This shift in position is something prettier would revert, I think. Try running it ?